src/toolbars/examples/toolbar.js

```diff
@@ -119,6 +119,7 @@
       <div class="col-md-12" ng-show="viewType == 'tableView'">
         <pf-table-view config="tableConfig"
                        colummns="colummns"
+                       colvis="colvis"
                        items="items"
```

Comment:
Couldn't we add the properties of 'colvis' to the already in use 'columns' object? Seems logical to add 'index' and 'visible' to columns obj. which already has other column attributes.